packages/formstr-app/src/containers/CreateFormNew/components/FormSettings/types.ts

```diff
@@ -8,4 +8,6 @@ export interface IFormSettings {
   formId?: string;
   encryptForm?: boolean
   viewKeyInUrl?: boolean
+  hideTitleImage?: boolean;
```

Comment:
why is this a prop in createForm? it is generally passed as a url parmater to form filler